app/src/main/java/at/aau/edu/wizards/api/impl/MessageDelegate.kt

```diff
@@ -13,6 +13,7 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import java.nio.charset.Charset
 
+// FIXME meaningful message content (eg, GameCommand), or message parsing in dedicated class for maintainability
```

Comment:
Note, before merging move somewhere else. MessageDelegate is decoupled from any specific parsing already

Reply:
Same fixme in GameModel::receiveMessage, but added here to consider everywhere else too